locales/nl/nl.json

```diff
@@ -266,8 +266,8 @@
     "If you did not request a password reset, no further action is required.": "Als je geen wachtwoordherstel hebt aangevraagd, hoef je verder niets te doen.",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Als je nog geen account hebt, kan je er een aanmaken door op onderstaande knop te klikken. Na het aanmaken van je account kan je op de uitnodiging in deze mail klikken om die te accepteren:",
     "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Is het nodig om specifieke contact- of BTW gegevens toe te voegen aan je bonnen, zoals je volledige bedrijfsnaam, BTW-nummer, of adress, dan kan je dat hier toevoegen.",
-    "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:",
```

Comment:
don't delete this line. the difference with the next line is just "you're" and "you’re"

Reply:
They are both present:
![image](https://user-images.githubusercontent.com/4425296/129159268-75053712-9e8b-45fe-9327-70c359a397db.png)